workitem/link/link_repository.go

```diff
@@ -279,3 +283,36 @@ func (r *GormWorkItemLinkRepository) Save(ctx context.Context, lt app.WorkItemLi
 	result := ConvertLinkFromModel(res)
 	return &result, nil
 }
+
+// ListWorkItemChildren get all child work items
+func (r *GormWorkItemLinkRepository) ListWorkItemChildren(ctx context.Context, parent string) ([]*app.WorkItem, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "workitem", "children", "query"}, time.Now())
+	wilt := WorkItemLinkType{}
+	r.db.Where("forward_name = ?", "relates to").First(&wilt)
```

Comment:
@kwk Ok, I am going to create a new WILT with the forward name as `parent of` and reverse name as `child of` under `system` category.